src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -185,6 +187,42 @@ public function table(array $headers, array $rows)
         $table->setHeaders($headers);
         $table->setRows($rows);
         $table->setStyle($style);
+        $table->setHorizontal($horizontal);
+
+        $table->render();
+        $this->newLine();
+    }
+
+    public function definitionList(...$list)
+    {
+        $style = clone Table::getStyleDefinition('symfony-style-guide');
+        $style->setCellHeaderFormat('<info>%s</info>');
+
+        $table = new Table($this);
+        $headers = [];
+        $row = [];
+        foreach ($list as $value) {
+            if ($value instanceof TableSeparator) {
+                $headers[] = $value;
+                $row[] = $value;
+                continue;
+            }
+            if (\is_string($value)) {
+                $headers[] = new TableCell($value, ['colspan' => 2]);
+                $row[] = null;
+                continue;
+            }
+            if (!\is_array($value)) {
+                throw new \InvalidArgumentException('Value should be an array, string or an instance of TableSeparator.');
```

Comment:
`, a string`

Reply:
also let's use the component's `InvalidArgumentException`

Reply:
Fixed, thanks